packages/x-components/src/x-modules/facets/components/filters/simple-filter.vue

```diff
@@ -29,6 +29,7 @@
         :disabled="isDisabled"
         data-test="filter"
         role="checkbox"
+        v-bind="$attrs"
```

Comment:
`v-bind="$attrs"` without `inheritAttrs: false` would still bind the attrs to the root element